.github/workflows/java-windows.yml

```diff
@@ -16,10 +16,17 @@ jobs:
                 uses: actions/setup-java@v1
                 with:
                     java-version: ${{ matrix.java_version }}
+            -   name: check dexer
+                run: Invoke-WebRequest -Uri "https://github.com/Sable/android-platforms/blob/master/android-28/android.jar?raw=true" -OutFile "./android.jar"
+                run: ./gradlew checkDexer
```

Comment:
seems like you cannot have multiple `run` in one step.

Reply:
yes I got to this error locally
do you know why is the travis queued for so long?

Reply:
it actually already failed: https://travis-ci.com/github/robstoll/atrium/builds/164298158
I guess the github did not get the notification